libwdi-sys/build.rs

```diff
@@ -281,13 +275,13 @@ impl LibwdiBuild
         // The source files that we'll copy as-is.
         // Again relative from libwdi_repo (after that .map() below, at least).
         let mut as_is: Vec<PathBuf> = [
+            "libwdi.c",
             "libwdi_dlg.c",
             "logging.c",
             "pki.c",
             "tokenizer.c",
             "vid_data.c",
             "embedder.c",
-            "installer.c",
```

Comment:
Was that removed on purpose? Just want to make sure. :)

Reply:
It was - the `as_is` array and the `needs_patch` array above are supposed to be mutually exclusive to each other (we hadn't realised this before) so as it's mentioned in `needs_patch` it must not be mentioned in `as_is`.

Reply:
Ok sounds good.